documentation/src/main/assembly/dist.xml

```diff
@@ -15,25 +15,68 @@
     <includeBaseDirectory>false</includeBaseDirectory>
 
     <fileSets>
-        <!-- Reference documentation -->
+        <!-- Reference documentation and getting started guides-->
+        <!-- -->
+        <!-- We want to preserve the same folder structure as it was generated by Asciidoctor -->
+        <!-- so that the links between the files would still work fine. -->
         <fileSet>
             <directory>${asciidoctor.aggregated-resources-dir}/css/</directory>
-            <outputDirectory>reference/en-US/html_single/css/</outputDirectory>
+            <outputDirectory>public/en-US/html_single/reference/css/</outputDirectory>
         </fileSet>
         <fileSet>
             <directory>${asciidoctor.aggregated-resources-dir}/script/</directory>
-            <outputDirectory>reference/en-US/html_single/script/</outputDirectory>
+            <outputDirectory>public/en-US/html_single/reference/script/</outputDirectory>
```

Comment:
If you're going to change the document structure, you will need to update some harcoded links.

Look for `https://docs.jboss.org` thoughought the codebase.

Reply:
I think the structure remained, with these latest changes, so no changes to doc URLs 🙈